canvas_modules/common-canvas/locales/command-actions/locales/en.json

```diff
@@ -16,6 +16,11 @@
 	"action.createSuperNode": "Create {node_label} supernode",
 	"action.deconstructSuperNode": "Deconstruct {node_label} supernode",
 	"action.deleteLink": "Delete link",
+	"action.deleteComments":"Delete {comments_count} comments",
+	"action.deleteNodes":"Delete {nodes_count} nodes",
+	"action.deleteLinks":"Delete {links_count} links",
+ 	"action.deleteCommentsLinks":"Delete {comments_count} comments and {links_count} links",
+	"action.deleteNodesLinks":"Delete {nodes_count} nodes and {links_count} links",
 	"action.deleteNodesCommentsLinks": "Delete {nodes_count} nodes, {comments_count} comments and {links_count} links",
 	"action.deleteNodesComments": "Delete {nodes_count} nodes, {comments_count} comments",
```

Comment:
Please change this message to: 
"Delete {nodes_count} nodes and {comments_count} comments"